setup.py

```diff
@@ -35,6 +35,7 @@ def get_install_requires() -> List[str]:
         "colorlog",
         "joblib",
         "numpy",
+        "redis",
```

Comment:
I actually think that `redis` is a rather trivial dependency compared to e.g. `alembic` which is used for our RDB storage. `redis` only uses the standard library.

Reply:
I agree with @c-bata because `RedisStorage` is still an experimental feature. I think that we can make `redis` as a required dependency when the feature become stable.

Reply:
So, you want me to make it extra requirement for now, and move it as requires dependency later?

Reply:
Added an experimental section in setup.py